.travis.yml

```diff
@@ -1,8 +1,20 @@
-language: node_js
 sudo: false
+
+git:
+  depth: 1
+
+branches:
+  only:
+    - master
```

Comment:
Result: It fails appropriately. 

I'll merge this (and squash down the commits) once we're green again